lib/karafka/server.rb

```diff
@@ -4,16 +4,15 @@ module Karafka
   # Karafka consuming server class
   class Server
     class << self
-      # We need to store reference to all the consumers in the main server thread,
-      # So we can have access to them later on and be able to stop them on exit
-      attr_reader :consumers
+      # Set of consuming threads. Each consumer thread contains a single consumer
+      attr_accessor :consumer_threads
```

Comment:
Why was this changed? Where is this used?

Reply:
It's not yet used directly as it only stores customer threads for reference. This will be used to #240